tests/test_compute_funs.py

```diff
@@ -413,3 +416,190 @@ def test_surface_areas():
+    # etc.
+
+    eqmmm = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=-1
+    )
+    eqmmp = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=-1
+    )
+    eqmpm = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=1
+    )
+    eqmpp = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=1
+    )
+    eqpmm = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=-1
+    )
+    eqpmp = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=-1
+    )
+    eqppm = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=1
+    )
+
+    eqppp = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=1
+    )
+
+    grid = Grid(np.array([[1, 0, 0]]))
+
+    # jacobian sign
+    assert np.sign(eqmmm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqmmp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqmpm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqmpp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqpmm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+    assert np.sign(eqpmp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+    assert np.sign(eqppm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+    assert np.sign(eqppp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+
+    grid = Grid(np.array([[0.1, 0, 0]]))
+    # toroidal field sign
+    assert np.sign(eqmmm.compute("B", grid=grid)["B"][0, 1]) == -1.0
+    assert np.sign(eqmmp.compute("B", grid=grid)["B"][0, 1]) == -1.0
+    assert np.sign(eqmpm.compute("B", grid=grid)["B"][0, 1]) == 1.0
+    assert np.sign(eqmpp.compute("B", grid=grid)["B"][0, 1]) == 1.0
+    assert np.sign(eqpmm.compute("B", grid=grid)["B"][0, 1]) == -1.0
+    assert np.sign(eqpmp.compute("B", grid=grid)["B"][0, 1]) == -1.0
+    assert np.sign(eqppm.compute("B", grid=grid)["B"][0, 1]) == 1.0
+    assert np.sign(eqppp.compute("B", grid=grid)["B"][0, 1]) == 1.0
+
+    grid = Grid(np.array([[1, 0, 0]]))
+    # poloidal field sign = -Bz on outboard side
+    assert np.sign(eqmmm.compute("B", grid=grid)["B"][0, 2]) == -1.0
+    assert np.sign(eqmmp.compute("B", grid=grid)["B"][0, 2]) == 1.0
+    assert np.sign(eqmpm.compute("B", grid=grid)["B"][0, 2]) == 1.0
+    assert np.sign(eqmpp.compute("B", grid=grid)["B"][0, 2]) == -1.0
+    assert np.sign(eqpmm.compute("B", grid=grid)["B"][0, 2]) == -1.0
+    assert np.sign(eqpmp.compute("B", grid=grid)["B"][0, 2]) == 1.0
+    assert np.sign(eqppm.compute("B", grid=grid)["B"][0, 2]) == 1.0
+    assert np.sign(eqppp.compute("B", grid=grid)["B"][0, 2]) == -1.0
+
+    grid = Grid(np.array([[0.1, 0, 0]]))
+    # toroidal current sign
```

Comment:
The same comment about "toroidal current" is used in another test below, although they are different physical quantities. It might be clearer to use names like "B_phi" and "G" in the comments.